source-asana/tests/snapshots/source_asana_tests_test_snapshots__capture__capture.stdout.json

```diff
@@ -216,11 +216,11 @@
         "gid": "1205846102972015"
       },
       "gid": "1205846058555984",
-      "html_text": "<b><a href=\"https://app.asana.com/0/profile/1205846104557056\">David Yaffe</a></b> changed the due date to Nov 2",
+      "html_text": "<b><a href=\"https://app.asana.com/0/profile/1205846104557056\">David Yaffe</a></b> changed the due date to Nov 2, 2023",
```

Comment:
lol I'm guessing it's just because the year rolled over.

Reply:
Oh duh, that makes perfect sense